src/App.js

```diff
@@ -181,23 +220,34 @@ const InnerNav = ({ client, setClient }) => {
   )
 }
 
-const Nav = ({ client, setClient }) => (
-  <NavigationContainer ref={RootNavigation.navigationRef}>
-    <SafeAreaProvider>
-      <ErrorProvider>
-        <LoadingOverlayProvider>
-          <OsReceiveProvider>
-            <InnerNav client={client} setClient={setClient} />
-          </OsReceiveProvider>
-        </LoadingOverlayProvider>
-      </ErrorProvider>
-      {client && <ClouderyOffer />}
-      <LockScreenWrapper />
-    </SafeAreaProvider>
-  </NavigationContainer>
-)
+const Nav = ({ client, setClient }) => {
+  const [markNameNav] = useState(() => rnperformance.mark('Nav'))
+  useEffect(() => {
+    rnperformance.measure({
+      markName: markNameNav,
+      measureName: 'Mount <Nav />'
+    })
+  }, [markNameNav])
```

Comment:
What do you think of an extracted hook like `usePerformanceMeasure(markName, mesureName)` ?

Reply:
Not sure how to implement it right now, so I propose to implement it in the future if we need it